packages/styles/src/components/header/_post-header.scss

```diff
@@ -180,14 +180,18 @@ post-header {
     }
   }
 
-  a.selected,
   post-togglebutton[aria-pressed='true'] {
     &:not(post-mainnavigation *) {
       color: color.$white;
       background: #050400;
     }
   }
 
+  a[aria-current='page'] {
```

Comment:
The selector was not working because the `a[aria-current]` is in the main navigation, and `&:not(post-mainnavigation *)` select only what's not in the main navigation.

Reply:
Because the `.selected` styles are not coming from this stylesheet but from the `post-mainnavigation`. Can you please merge the selectors?